src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/get-started.mdx

```diff
@@ -0,0 +1,412 @@
+
+Once you have the Lambda CLI you can install the serverless framework. This is the foundation for containerized functions. Run the following script:
+
+```
+npm install -g serverless
+
+```
+</Step>
+<Step>
+### Install the New Relic serverless framework [#install-new-relic-serverless-framework]
+
+After you've installed Lambda's serverless framework, you need to install New Relic's serverless framework. Complete the following steps:
+
+1. Run: 
+
+```
+npm install --save-dev serverless-newrelic-lambda-layers
+```
+2. Then run:
+
+```
+npm add --dev serverless-newrelic-lambda-layers
+
+```
+</Step>
+<Step>
+## Add the plugin [#plugin]
+
+You can choose any example code in the `serverless-newrelic-lambda-layers` repository. Once you've selected the code, edit the `serverless.yml` file and add the plugin using the following steps:
+
+```
+plugins: 
+- serverless-newrelic-lambda-layers
+```
+You can set the name using the service name function in the `serverless.yml` file.
+
+```
+service: <function name>
+```
+
+The following is an example:
+
+```
+custom:
+  newRelic:
+    accountId: <accountId>
+    apiKey: <apiKey>
+    enableIntegration: true
+            enableFunctionLogs: true
+    enableExtension: true
+    logLevel: debug
+    linkedAccount: New Relic Lambda Integration - <accountId>
+```
+</Step>
+<Step>
+## Deploy [#deploy]
+You have now completed all the necessary steps to create a containerized function. All you have to do now is deploy it by running the following script:
+
+```
+sls deploy
+```
+
+After you've successfully deployed the containerized function, you can test the function using the following command:  
+
+```
+newrelic-lambda functions list
+
+```
+</Step>
+<Step>
+## Add New Relic layers [#add-new-relic-layers]
+
+Now that you've successfully deployed, you can create layers and add subscriptions. You can do this by running the following commands:
+
+```
+newrelic-lambda layers install --nr-account-id <accountId> --function <function name>
+```
+And:
+
+```
+newrelic-lambda subscriptions install --function <funcion name>
+```
+</Step>
+</Steps>
+
+Keep in mind that New Relic for Node.js automatically instruments most standard web requests, but occasionally you might want expanded instrumentation. Using the agent's custom instrumentation API, you can create instrumentation for otherwise unsupported web frameworks, data stores, and message service clients.
+</Collapser>
+</CollapserGroup>
+
+<CollapserGroup>
+ <Collapser
+   id="node-18"
+   title="Create a Lambda containerized function with the Node.js 18. x agent"
+ >
+## Requirements [#Requirements]
+
+To create a Lambda containerized function using the Node18.x agent, you will need the following:
+
+* [AWS Command Line Interface (AWS CLI) version 2] (https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+* [Docker ](https://docs.docker.com/get-docker/)
+* Node.js
+
+<Steps>
+<Step>
+## Create a project directory [#project-directory]
```

Comment:
It should be something like: Create a sample containerized lambda function with node